### PR TITLE
Add "Pull Request Ratio Flow" insight chart

### PR DESCRIPTION
**Pull request by se7entyse7en**
Closes [#362](https://athenianco.atlassian.net/browse/ENG-362).
Depends on: #132, #136, #138, #139.

## Screenshot
Screenshot 2020-03-25 at 16 43 25

### CODE REVIEW

File: src/js/components/insights/stages/WorkInProgress.jsx [hunk 1]
```diff
@@ -1,12 +1,147 @@
...
 
 export const getCharts = () => [
-    mostActiveDevs
+    mostActiveDevs,
+    prRatioFlow
 ];
 
+const prRatioFlow = {
+    fetcher: async (api, context, data) => {
+        const metricIDs = new MetricID();
```
Comment (dpordomingo) [inline src/js/components/insights/stages/WorkInProgress.jsx]: the very concern I already have at https://github.com/athenianco/athenian-webapp/pull/139#discussion_r398493164

**Comment by se7entyse7en** [comment]
@warenlg 

Screenshot 2020-03-26 at 13 39 57

**Comment by warenlg** [comment]
Very nice chart!! we clearly see the weekends!
However, the 2nd KPI is suspicious to me: how can we have an avg PR ratio flow for Mondays of 3 when the max seems to be 2 on Feb 14th?

**Comment by se7entyse7en** [comment]
Good point. The reason is this:

Screenshot 2020-03-26 at 13 53 31
Screenshot 2020-03-26 at 13 53 37

So basically `Monday` in the chart do not appear because `pr-flow` is `0` since denominator is `0`. But the overall KPI what does is `sum(opened) / sum(closed)` by day, so for `monday` is `3` even if the single datapoints are all zeros.

Do you think is better to change it to the average of the `pr-flow`?

**Comment by se7entyse7en** [comment]
This has been solved on Slack. Each datapoint is calculated as `(opened || 1) / (closed || 1)` and the for a day we compute the average of the datapoints.

**Pull request merged**